packages/web-components/src/components/ic-top-navigation/ic-top-navigation.tsx

```diff
@@ -312,17 +314,30 @@ export class TopNavigation {
             <header role="banner">
               <div class="top-panel-container">
                 <div class="app-details-container">
-                  {hasTitle && (
-                    <a class="title-link" href={this.href}>
+                  {isSlotUsed(this.el, "router-app-title") ? (
+                    <div class="title-link">
                       {this.hasAppIcon && (
                         <div class="app-icon-container" aria-hidden="true">
                           <slot name="app-icon" />
                         </div>
                       )}
                       <ic-typography variant={appTitleVariant}>
-                        <h1 class="title-wrap">{this.appTitle}</h1>
+                        <slot name="router-app-title"></slot>
```

Comment:
Is the expectation here then that when they use a router, they need to add the relevant heading tag like h1 themselves? May have read this incorrectly so apologies if not. I'm curious if so though, do we have a ticket where we'll be documenting on the website how to use a router with the app title, where we can make sure this specific point is mentioned?

Reply:
The appTitleVariant will control the relevant heading tag, so the user won't need to change anything :)

Reply:
We'll still need a ticket to show how to use app title with React router in the website though right. Let me know if this exists already or if you were creating it

Reply:
I'll add a ticket :)